resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesUtils.scala

```diff
@@ -381,4 +381,34 @@ object KubernetesUtils extends Logging {
       }
     }
   }
+
+  /**
+   * This function builds the EnvVar objects for each key-value env.
+   */
+  @Since("3.4.0")
+  def buildEnvVars(env: Map[String, String]): Seq[EnvVar] = {
+      env.filter( env => env._2 != null)
+         .map { env =>
+           new EnvVarBuilder()
+             .withName(env._1)
+             .withValue(env._2)
+             .build()
+      }.toSeq
+  }
+
+  /**
+   * This function builds the EnvVar objects for each field ref env.
+   */
+  @Since("3.4.0")
+  def buildEnvVarsWithFieldRef(env: Seq[(String, String, String)]): Seq[EnvVar] = {
+      env.filter( env => env._2 != null && env._3 != null)
```

Comment:
ditto. Two-space indentation and removal of space by `( end =>` -> `(end =>`.

Reply:
updated